recipes/hiproc/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "hiproc" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7a25cbf8d054fe1caa6d69c5339ed4248ac733f34f69b83fe7c96f6bd3366e7c
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

It's fine if windows then fails the CI

---

Yes, I put in the `skip: True  # [win]` to avoid the windows fail (and then had to remove the `noarch: python` but I can put them both back in, if that's preferred.